scripts/upgrade/readarr.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+if [[ ! -f /install/.readarr.lock ]]; then
+    echo_error "Readarr not detected. Exiting!"
+    exit 1
+fi
+
+box install readarr
```

Comment:
If readarr can self-update then I don't think we really need it at all now that I think of it

Reply:
Would be good to verify that readarr _can_ self update properly and that no permission issues are present

Reply:
Per PearsonFlyer on Discord, seems so:
```I updated to .330, and it's been auto-updating since then with no issues.
(330 manually installed, then auto-update, currently on .338)```

Reply:
With that i meant more that the paths that Readarr uses to update itself like the folders in `/tmp` have the right permissions, and everything it should own in its `/opt` folder

Reply:
Readarr is built like Radarr (v3) and Lidarr (v0.8), so it should (in theory) be able to update itself

Reply:
Then an upgrader is not necessary at all.